lib/actions/ui.js

```diff
@@ -1,4 +1,6 @@
-import {EOL} from 'os';
+import os from 'os';
+import shell from 'electron';
```

Comment:
`...showItemInFolder is not a function` is thrown because of this line. This isn't importing the way you think it is, this should be `import {shell} from 'electron';`. Right now, it is importing the default electron export, which doesn't lineup to what you are calling below.